ckan_cloud_operator/providers/ckan/db/migration.py

```diff
@@ -81,12 +81,12 @@ def delete(name, delete_dbs=False):
     migration = crds_manager.get(CRD_SINGULAR, name=name, required=False) or {}
```

Comment:
why is this file being modified?

Reply:
Azure SQL requires username in `username@host` format. Connection string with two `@` is not supported in psycopg2, so we pass connection parameters as kwargs.

Reply:
The question is what is db_migration being used for in the context of Azure?

Reply:
It has been used when I set it up with deis instances. It's not used anymore. Reverting it.